static/openapi/prerendering-spec.json

```diff
@@ -1989,11 +1987,195 @@
         }
       }
     },
-    "/api/prerender/v1/origins/{_key}": {
+    "/api/prerender/v1/origins/{origin}": {
+      "get": {
+        "summary": "Get origin",
+        "tags": ["Origins"],
+        "description": "This endpoint returns an origin from the origin-related GDN fabric.",
```

Comment:
Do you really want to talk about GDN in photoniq documentation?  Can you word this without using GDN?